packages/pretty-format/src/plugins/ReactElement.js

```diff
@@ -121,6 +124,9 @@ const print = (
   return result;
 };
 
+// Disabling lint rule as we don't know type ahead of time.
+/* eslint-disable flowtype/no-weak-types */
 const test = (object: any) => object && object.$$typeof === reactElement;
```

Comment:
I assume since this is a test method we don't want to explicitly set the `object` type to `React$Element<any>`?